app/forms/new_phone_form.rb

```diff
@@ -180,4 +182,29 @@ def ingest_submitted_params(params)
   def confirmed_phone?
     false
   end
+
+  def validate_phone_submission_limit
+    fingerprint = Pii::Fingerprinter.fingerprint(Phonelib.parse(phone).e164.to_s)
+    @submission_rate_limiter ||= RateLimiter.new(
+      target: fingerprint,
+      rate_limit_type: :phone_fingerprint_confirmation,
+    )
+    @submission_rate_limiter.increment!
+    lock_out_phone_fingerprint if @submission_rate_limiter.maxed?
+  end
+
+  def lock_out_phone_fingerprint
+    errors.add(
+      :phone_fingerprint,
```

Comment:
I like this new validation approach! Much simpler.

As mentioned in https://github.com/18F/identity-idp/pull/10459#discussion_r1583153536, the phone setup controller won't surface errors by default, so we'll need to revise that code if we want this error message to be visible to the user. This also seems like something we'd want to test for in a feature spec.

I could imagine the suggestion in https://github.com/18F/identity-idp/pull/10459#discussion_r1583153536 could be implemented something like...

```rb
class FormResponse
  # ...
  def first_error_message(*keys)
    return if errors.blank?
    [*keys, errors.keys.first].each do |key|
      return errors[key].first if errors.key?(key)
    end
  end
end
```